scenario/serializers.py

```diff
@@ -105,3 +108,25 @@ class ModuleScenarioSerializer(serializers.ModelSerializer):
     class Meta:
         model = Module
         fields = ['id', 'request', 'x_position', 'y_position']
+
+
+class ScenarioHistorySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ScenarioHistory
+        fields = '__all__'
+
+    def validate(self, attrs):
+        print("Hello")
+        if len(attrs['order']) == 0 and ('schedule' not in attrs):
+            raise serializers.ValidationError('you should pass order or schedule')
+        if len(attrs['order']) != 0:
+            for o in attrs['order']:
+                get_object_or_404(RequestHistory, id=o)
```

Comment:
inja ham nabas get_object_or_404 bashe, in faqat tu viewwww 😂😂😂😂

Reply:
hallllllllllllllll